docs/markdown/snippets/int_to_string_fill.md

```diff
@@ -0,0 +1,17 @@
+## Added 'fill' kwarg to int.to_string()
+
+int.to_string() now accepts a `fill` argument. This allows you to pad the
+string representation of the integer with leading zeroes:
+
+```meson
+message(4.to_string())
+message(4.to_string(length: 3))
+message(4.to_string(length: -3))
```

Comment:
Small nitpicking, I think the example with -4 is more useful than with negative length. Other than that, LGTM.

Reply:
Updated